dev/PipsPager/PipsPager.cpp

```diff
@@ -86,12 +86,16 @@ void PipsPager::OnApplyTemplate()
     }(GetTemplateChildT<winrt::Button>(c_nextPageButtonName, *this));
 
     m_pipsPagerElementPreparedRevoker.revoke();
+    m_pipsAreaGettingFocusRevoker.revoke();
+    m_pipsAreaBringIntoViewRequestedRevoker.revoke();
     [this](const winrt::ItemsRepeater repeater)
     {
         m_pipsPagerRepeater.set(repeater);
         if (repeater)
         {
             m_pipsPagerElementPreparedRevoker = repeater.ElementPrepared(winrt::auto_revoke, { this, &PipsPager::OnElementPrepared });
+            m_pipsAreaGettingFocusRevoker = repeater.GettingFocus(winrt::auto_revoke, { this, &PipsPager::OnPipsAreaGettingFocus });
+            m_pipsAreaBringIntoViewRequestedRevoker = repeater.BringIntoViewRequested(winrt::auto_revoke, { this, &PipsPager::OnPipsAreaBringIntoViewRequested });
```

Comment:
BringIntoViewRequested is only available in 1803 and upwards (RS4+), as such this crashes the test app on RS2 and RS3 as it can't find that event. Given that was designed to be our downlevel workaround, I don't think we have many other options left beside just letting it be that way.